src/collections/_documentation/workflow/releases/health.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Health (Beta)
+sidebar_order: 1
+---
+
+{% capture __alert_content -%}
+Sentry’s Release Health features are currently in beta. For more details about access to these features, feel free to reach out to mobile@sentry.io.
+{%- endcapture -%}
+{%- include components/alert.html
+    title="Note"
+    content=__alert_content
+    level="warning"
+%}
+
+Monitor the health of releases by observing user adoption, usage of the application, percentage of [crashes](#crash), and [session data](#session). Release health will provide insight into the impact of crashes and bugs as it relates to user experience, and reveal trends with each new issue through the release details, graphs, and filters.
+
+## How Release Health Works
+
+Once configured, Sentry's mobile and browser SDKs will send health data about the usage of your application.
+
+The primary component that Sentry uses to monitor health is a session. A session represents the interaction of the user with the application and starts when a user opens the application. When a user closes the application or has a prolonged period of inactivity, the session ends.
+
+Sessions are submitted to Sentry so you can track the usage and adoption of your application. When a user of your application experiences a crash, error, or abnormal exit, the session will be flagged accordingly, and Sentry calculates derived metrics. The metrics include data such as the number of users that didn't experience a crash in the specified time range. Sentry connects the metrics to the specific version of your application and the associated code.
```

Comment:
`to the specific version of your application` mmm, just wondering as one can have a version`x.y.z` but a different `patch` version if using `semver`. Should we use the `version` wording or `release` version? not sure if it makes any difference tbh.

<img width="355" alt="Screenshot 2020-04-16 at 09 21 00" src="https://user-images.githubusercontent.com/5731772/79426598-a7df4900-7fc3-11ea-9253-48c560c6df68.png">

Reply:
I would not mention it in  this section since the  goal of the section is only to provide high level overview.

Reply:
Updated.